services/consensuscontext/validate_transactions_block.go

```diff
@@ -205,12 +230,30 @@ func validateTransactionsBlockTxTriggerIsValid(signedTransaction *protocol.Signe
 
 func (s *service) ValidateTransactionsBlock(ctx context.Context, input *services.ValidateTransactionsBlockInput) (*services.ValidateTransactionsBlockOutput, error) {
 	vctx := &txValidatorContext{
-		protocolVersion:        s.config.ProtocolVersion(),
 		virtualChainId:         s.config.VirtualChainId(),
 		allowedTimestampJitter: s.config.ConsensusContextSystemTimestampAllowedJitter(),
 		input:                  input,
 	}
 
+	ref, err := s.management.GetCurrentReference(ctx, &services.GetCurrentReferenceInput{})
+	if err != nil {
+		s.logger.Error("management.GetCurrentReference should not return error", log.Error(err))
+		return nil, err
+	}
+	if err2 := validateProposeBlockReferenceTime(input.PrevBlockReferenceTime, input.TransactionsBlock.Header.ReferenceTime(),
```

Comment:
do we need the errors later? a little strange to see the index numbers on the errors

Reply:
sometimes the bot hates it when you reuse error, so i changed to have errX.